lms/static/js/student_account/utils.js

```diff
@@ -25,8 +25,13 @@
                 }
 
                 cookie = document.cookie.match('(^|;)\\s*' + edxUserCookie + '\\s*=\\s*([^;]+)');
+                console.log("POUAC", cookie)
```

Comment:
I don't know much about JavaScript tests either. Did this `console.log` ever show up in your test run?